### ISSUE TITLE
Similar images will generate the same MD5 file name

### ISSUE THREAD

**Issue by ln0y** [Sergei-Korneev/obsidian-local-images-plus#11]
!\[img\](https://p3-juejin.byteimg.com/tos-cn-i-k3u1fbpfcp/f6f0a43c7a1c407ab75b3bbb37c0dc63~tplv-k3u1fbpfcp-zoom-1.jpeg)
!\[img\](https://p3-juejin.byteimg.com/tos-cn-i-k3u1fbpfcp/277c7243c88a4710aa6aaf5b8a269f3c~tplv-k3u1fbpfcp-zoom-1.jpeg)

I think it's because the slice is too short.

https://github.com/Sergei-Korneev/obsidian-local-images-plus/blob/e93fa64862ff94ac09d69b96c9778f6fab335243/src/contentProcessor.ts#L181

nodejs v18.13.0
```js
!(async function () {
  const enc = new TextDecoder('utf-8')

  const file1 = await fetch(
    'https://p3-juejin.byteimg.com/tos-cn-i-k3u1fbpfcp/f6f0a43c7a1c407ab75b3bbb37c0dc63~tplv-k3u1fbpfcp-zoom-1.jpeg',
    { responseType: 'arraybuffer' }
  ).then(res => res.arrayBuffer())

  const code1 = enc.decode(file1.slice(0, 15000))

  const file2 = await fetch(
    'https://p3-juejin.byteimg.com/tos-cn-i-k3u1fbpfcp/277c7243c88a4710aa6aaf5b8a269f3c~tplv-k3u1fbpfcp-zoom-1.jpeg',
    { responseType: 'arraybuffer' }
  ).then(res => res.arrayBuffer())

  const code2 = enc.decode(file2.slice(0, 15000))

  console.log(code1 === code2) // true
})()
```

**Comment by Sergei-Korneev** [comment]
You know, that is a fair observation. A good one, thanks. Will fix this in 0.14.9 . Also the arraybuffer may vary within wide limits. So this part of code should be rewritten.  Nice avatar, btw ;)

**Comment by ln0y** [comment]
haha, thank you, I'm looking forward to it!
